src/test/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/integration/CancelVisitTest.kt

```diff
@@ -0,0 +1,175 @@
+package uk.gov.justice.digital.hmpps.visitscheduler.integration
```

Comment:
So much better

Reply:
Yep going to try and slowly break up that big integration test class